### ISSUE TITLE
Add training mode for OLL and PLL

### ISSUE THREAD

**Issue by jayp33** [hakatashi/smart-cube-timer#33]
I am learning OLL and PLL, so I thought that it would be very useful if the timer had an option to show which algorithms should be used after completing F2L. The 2-Look method should suffice.

**Comment by cs0x7f** [comment]
Well, I might misunderstand you. Do you mean that, let the timer tell you how to solve the oll and pll when using a Giiker cube?

**Comment by jayp33** [comment]
Yes, the idea was for it to show the complete algorithm. But I didn't realize until now that it actually already shows the name of the algorithm to use after completing F2L.

**Comment by cs0x7f** [comment]
Well, for each OLL or PLL, there will be multiple algorithms (http://algdb.net/puzzle/333/pll). It's hard for the timer to decide which to show.

**Comment by jayp33** [comment]
Hmm, true. The timer needs a database of the most common algorithms for each OLL and PLL. The timer should have a default but also let the user pick other algorithms for each OLL and PLL in a settings menu. This is quickly becoming a bigger feature than I expected. ;)

**Comment by cs0x7f** [comment]
And another question is, after F2L, the orientation of the cube is not determined. How does the timer knows the orientation of the physical cube?

**Comment by jayp33** [comment]
Yes, this also occurred to me. The timer would need to indicate the color of the face that you're supposed to be looking at, so you can do a cube rotation.
